src/components/jnlp-page/Hero-JNLP.astro

```diff
@@ -24,29 +24,29 @@ import screenshots from "@/components/jnlp-page/assets/screenshots.png";
 					<span class="font-bold text-orange-500"> Java Web Start</span>
 					<br />apps in the browser
 				</h1>
-				<p class="text-xl mt-8 text-stone-400 leading-7">
+				<p class="text-xl mt-12 text-stone-300 font-semibold leading-7">
 					Unleash the power of CheerpJ in an easy-to-use browser extension!
 				</p>
-				<p class="text-xl mt-8 text-stone-400 leading-7">
+				<p class="text-xl mt-8 text-stone-300 leading-7">
 					Run Java Web Start Applications, including Oracle E-Business Suite and
 					Oracle Forms Applications, without a Java Desktop installation.
 				</p>
 				<div class="mt-8 flex flex-wrap gap-4">
 					<LinkButton
-						href="https://chromewebstore.google.com/detail/cheerpj-jnlp-runner/achnhggjjhnadfapkmgallnbmemdojdi"
-						label="Add to Chrome"
+						href="/cheerpj-jnlp-runner/getting-started/install-chrome"
+						label="Install for Chrome"
 						type="primary"
 						iconRight="ri:chrome-fill"
 					/>
 					<LinkButton
-						href="https://microsoftedge.microsoft.com/addons/detail/cheerpj-jnlp-runner/dpleickbpchmkpkmicanfolmianepfhe"
-						label="Add to Edge"
+						href="/cheerpj-jnlp-runner/getting-started/install-edge"
+						label="Install for Edge"
 						type="primary2"
 						iconRight="ri:edge-fill"
 					/>
 					<LinkButton
-						href="/cheerpj-jnlprunner"
-						label="Read the docs"
+						href="/cheerpj-jnlp-runner/getting-started"
+						label="Documentation"
```

Comment:
I see what you did here! Very clever! It didn't occurred to me, I like this overlap of product page with the docs overview, avoids information from being redundant. 

Just one thing, in my screen size (Macbook pro 13") the slider button and the "Read the docs" button are very close. When I'm at the top of the page and try to click on "read the docs" it clicks on the slider instead. It may be good to space out one from the other to avoid this.